modules/seqexec/server/src/main/scala/seqexec/server/ghost/Ghost.scala

```diff
@@ -42,6 +43,10 @@ final case class Ghost[F[_]: Sync](controller: GhostController[F])
   override val observeControl: InstrumentSystem.ObserveControl =
     InstrumentSystem.Uncontrollable
 
+  override def bundleKeywords(
+    ks: List[KeywordBag => SeqActionF[F, KeywordBag]]): SeqActionF[F, KeywordBag] =
+    GdsInstrument.bundleKeywords(ks)
```

Comment:
Is this because right now, GHOST doesn't have any other keywords explicitly defined?